src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/ClientFileMetadataFields.scala

```diff
@@ -0,0 +1,40 @@
+package uk.gov.nationalarchives.tdr.api.graphql.fields
+
+import io.circe.generic.auto._
+import sangria.macros.derive._
+import sangria.marshalling.circe._
+import sangria.schema.{Argument, Field, InputObjectType, ObjectType, fields}
+import uk.gov.nationalarchives.tdr.api.graphql.ConsignmentApiContext
+import uk.gov.nationalarchives.tdr.api.graphql.fields.FieldTypes._
+
+object ClientFileMetadataFields {
+  case class ClientFileMetadata(fileId: Long,
+                                originalPath: Option[String] = None,
+                                checksum: Option[String] = None,
+                                checksumType: Option[String] = None,
```

Comment:
This reminds me: do you know what checksum algorithms we're going to use in the frontend? Are we planning to just use one algorithm? Just wondering if this DB column still makes sense.

Reply:
No idea. Is it likely we would change checksum algorithm at some point? If so this column would still make sense

Reply:
Sorry, I meant that if we were using more than one algorithm then we'd want to structure the data differently.

I'll check with David to make sure he's still happy with the algorithm we used in the prototype.